### DISCUSSION TITLE
Custom filters in admin list view

### DISCUSSION THREAD

**Discussion by SimonvanDoorne** [joomengine/Joomla-Component-Builder#760] [Q&A]
Hi there,
I need to make a set of filters in my admin view and i haven't found an easy way to do so. Here's what i need:

- I have a page where i list some products, and some products have a filled in Product ID, while others do not. I need a filter so you can filter on all the products that have a Product ID that's filled.
- A filter where you can select 0 and 1 (with preferably yes and no as text) to filter a tinyint field on those values.

I remember a while ago seeing a mail or discussion about custom filters but for the life of me i can't seem to find it again.
Thanks in advance!

**Comment by Llewellynvdm** [comment]
So I am planning to explain this in the next JUG meeting the 8th of July, hope to see you there.
